_posts/2018-09-17-not-your-kind-of-automation.md

```diff
@@ -0,0 +1,103 @@
+</center>
+
+Just think of how much time everyone saves with just this one feature.
+
+Currently it also has some basic statistics, automate a hell load of features, it’s performant, has some easter eggs to be found and brings new technologies that can serve for something else, and, it won’t stop growing.
+
+Below I share current major features:
+
+- Basic statistics;
+- Amount of new customers, new orders, sum of value of orders;
+- Average orders per minute;
+- Orders placed by device and payment method;
+- Configuration search and audit (like a history of changes);
+- Moving order status on test environments;
+- Several syncs and resyncs between systems, like items, orders, stock, suppliers and others;
+- Export of vouchers, customers, vendors and others;
+- Overview of what is deployed on each test environment with several details like, when, git revision and others;
+- Debugger for Android/iOS apps requests;
+- And much more...
+
+## Impact and how we made it possible
+
+With such initiatives you get the attention of everyone, all the teams participate in order to improve what you’ve built, give feedback and suggestions. These kind of tools, empower people and make their life’s easier since every task is performed with efficiency and speed. On my team I do promote a simple Hackathon on every first day of a sprint. It’s here where new ideas come to daylight, just like this one did.
+So, I do encourage you to do the same. Even with multidisciplinary teams, keeping high performance, without losing productivity and not abandoning your roadmap, it’s possible to do it, just plan it, organize your days, delegate and lead, trust me, everyone can do it.
+
+## Conclusion
+
+Apart from the obvious benefits of automating your daily, repetitive tasks, it has a huge impact on your team and your peers, and below are some key points:
+
+- Involvement;
+- Empowering;
+- Peer recognition;
+- Motivation;
+- Inspiration;
+
+With it, other benefits also arise. The team is able to work on something new, imagine, eating the same thing every day, yeah, you will be tired of it and may never eat it again. The same happens with technologies, and, developers are thirsty for new things. But that’s not all, in this blog post, as Team Leader, I do want you, that have a leader role to involve your team members in the creation of tools that ease everyone’s job, do regular small Hackathons with them, use this opportunity to use new technologies, motivate along the way, let them grow, give them space to experiment, take risks and do mistakes. You’ll see that only good things will show up.
+
+## About Me
+
+I’m Nuno Santos, started as developer at Jumia in 2012 and have been invited to be a Team Leader in 2015 and founded Ninja Bytes along with it’s members. A team that i’m proud of, being bold and innovative, brings innovation all the times and is always one step ahead leading Jumia to success.
```

Comment:
> Nuno Santos, started as developer at Jumia in 2012 

Nuno Santos, started as **a** developer at Jumia in 2012 

> A team that i’m proud of, being

A team that **I’m** proud of, being

Reply:
Thanks, nice catch!